src/routes/docs/tooling/command-line/non-interactive/+page.markdoc

```diff
@@ -16,9 +16,19 @@ When you set the global configuration parameters using the `appwrite client` com
 
 In this mode, the CLI can only interact with one project at a time.
 
+To run tests or project-per-branch previews, you will need Console SDK capabilities. To accomplish this, log in using the `appwrite login` and pass in your email, password, and OTP settings as parameters.
+
+```sh
+appwrite login --email "<EMAIL>" --password "<PASSWORD>" --mfa totp  
+```
+
 # API Keys {% #api-keys %}
 In non-interactive mode, the CLI uses an API key to authenticate. Your API key must have sufficient permissions to execute the commands you plan to use. [Learn more about API Keys](/docs/advanced/platform/api-keys).
 
+{% info title="Using API keys with CLI" %}
+Some CLI commands won't work with an API key, like the `projects` commands.
+{% /info %}
+
```

Comment:
The whole beginning up to here is quite convoluted and confusing now because it jumps back and forth between multiple authentication methods. How can all of this be re-written to flow better?

Reply:
It's still pretty convoluted, jumping between API key, email and password, and then back to API key. Keep in mind, only one approach is used at a time.

How about we reorganize this to

```
The Appwrite CLI can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server.

# Authentication

<!--  small paragraph to introduce the two methods and the difference (in particular API key won't work for project api calls -->

## Email and password

[...]

## API keys

[...]

```

Reply:
I meant the whole beginning including "The Appwrite CLI can be used in a non-interactive and headless manner" at the top of the page.

<img width="743" alt="image" src="https://github.com/user-attachments/assets/e0167c87-e949-486e-b297-92a6bdd7f8d4">